hang_man/main.js

```diff
@@ -0,0 +1,114 @@
+const mystery_words_array = [
+  "Malamute",
+  "Poodle",
+  "Labrador",
+  "Spaniel",
+  "Bloodhound",
+  "Sheppard",
+  "Weiner",
+  "Bulldog",
+  "Boxer",
+  "Chihuahua",
+  "Mutt",
+  "Weimaraner",
+  "Pug"
+]
+const mystery_word = mystery_words_array[Math.floor(Math.random()*mystery_words_array.length)].toLowerCase();
```

Comment:
This should probably be a function (`getRandomWord`).

Also, given that you're calling `toLowerCase` regardless, you might as well just store your words in lowercase in the array to begin with.